extensions/bitwarden/src/api.ts

```diff
@@ -114,6 +115,30 @@ export class Bitwarden {
     return stdout;
   }
 
+  async copyPasteUsernamePassword(password: string, username?: string): Promise<void> {
+    osascript.execute(`
+      tell application "System Events"
+      set textToType to username
+      delay 0.2
+      repeat with char in (textToType)
+        keystroke char
+        delay 0.05
+      end repeat
+      keystroke tab
+      set textToType to passwordText
+      delay 0.2
+      repeat with char in (textToType)
+        keystroke char
+        delay 0.05
+      end repeat
+      delay 0.2
+      key code 36
+      end tell`,
```

Comment:
I don't think this is robust enough to cover all of the usecase. What if the input website does not support tab navigation ? 

The bitwarden extension is working just fine for this usecase, and is probably analyzing the html code to find out what input to fill.
You can even assing a shortcut to fill the form automatically.